Warn that PDF generation is not possible packaged gnuplot, due to the
missing pdflib (closes: #478677)

 -- Thomas Weber <thomas.weber.mail@gmail.com>  Sat, 24 May 2008 13:49:51 +0200

--- a/scripts/plot/print.m
+++ b/scripts/plot/print.m
@@ -141,6 +141,9 @@
 ##
 ##   @item pdf
 ##     Portable document format
+##
+##     NOTE: The gnuplot binary as shipped by Debian cannot create PDF files, see http://bugs.debian.org/478677
+##
 ##   @end table
 ##
 ##   If the device is omitted, it is inferred from the file extension,
